conda/base/constants.py

```diff
@@ -90,6 +90,8 @@
 
 UNKNOWN_CHANNEL = "<unknown>"
 
+PIP_PSEUDOCHANNEL = "<pip>"
```

Comment:
@kalefranz we'll want to talk about this... this was prompted by the fact that `egg_info` needed to return `IndexRecord` objects to accomplish this goal... and that has some interesting potential future possibilities, such as giving `conda` the ability to replace a `pip` package with `conda` package...